### DISCUSSION TITLE
alist用一会就会断开，显示未运行，开关启用也不行，只能重启openwrt

### DISCUSSION THREAD

**Discussion by delicioushotpot** [AlistGo/alist#5537] [General]
![image](https://github.com/alist-org/alist/assets/61256807/d5ad2772-ba85-44bd-81fe-701912672050)

**Comment by tuweiping001** [comment]
同样的问题，有解决吗？

**Reply by Lostfairy2023** [reply]
看楼下 同样的问题 已经解决！

**Comment by rzchoi** [comment]
https://github.com/sbwml/luci-app-alist

How to install prebuilt packages
Login OpenWrt terminal (SSH)

Install curl package
opkg update
opkg install curl

Execute install script (Multi-architecture support)
sh -c "$(curl -ksS https://raw.githubusercontent.com/sbwml/luci-app-alist/master/install.sh)"

**Reply by Lostfairy2023** [reply]
感谢大神！！完美解决问题！！！
